serenata_toolbox/reimbursements.py

```diff
@@ -41,7 +41,7 @@ def read_csv(self, name):
     @property
     def receipts(self):
         print('Merging all datasets…')
-        datasets = ('current-year.xz', 'last-year.xz', 'previous-years.xz')
+        datasets = ["reimbursements-{}.xz".format(n) for n in range(2009, date.today().year+1)]
```

Comment:
I'm not a big fan of long lines. If I was coding that probably I would extract it as a method decorated with `@property`:

```python
class Reimbursements:

    …
    @property
    @staticmethod
    def datasets():
        for year in range(2009, date.today().year+1):
            yield "reimbursements-{}.xz".format(year)
```